app/src/androidTest/java/org/simple/clinic/overdue/download/OverdueListDownloaderIntegrationTest.kt

```diff
@@ -34,7 +34,7 @@ class OverdueListDownloaderIntegrationTest {
   fun downloading_a_csv_should_work_correctly() {
     // when
     val result = overdueListDownloader
-        .download(OverdueListFileFormat.CSV)
+        .download(OverdueListFileFormat.CSV, listOf(UUID.fromString("762f1a02-76aa-4dd7-8669-48aa0deac64b")))
```

Comment:
I didn't add the selected ids here as there are couple of new tests that handle ids. Here I was verifying it loads from the server API. So, pass an empty list here?